MatchZy.cs

```diff
@@ -120,6 +122,28 @@ public partial class MatchZy : BasePlugin
                 { ".exitprac", (player, commandInfo) => OnMatchCommand(player, commandInfo) },
                 { ".stop", (player, commandInfo) => OnStopCommand(player, commandInfo) }
             };
+            
+            RegisterListener<Listeners.OnClientConnected>((slot =>
+	        {
+		        var player = Utilities.GetPlayerFromSlot(slot);
+		        if(player.IsBot) return;
+		        var steamId = player.SteamID;
+		
+		        string whitelistfileName = "puggr/whitelist.cfg";
```

Comment:
Hey!
Instead of using `puggr` folder, can we use `MatchZy` folder name?
Because all the cfgs related to this plugin are stored in cfg/MatchZy/

Reply:
oh yeah i forgot to change that folder structure to MatchZy since ive been using that whitelist function in a different plugin. Ive pushed a fix